src/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.cpp

```diff
@@ -587,71 +660,28 @@ File Manager::getLocalRepositoryLockFile() const
 File Manager::getLatestBulletinAFile() const
```

Comment:
This function and the one below change due to removing the dated subdirectory for BulletinA and Finals2000A (e.g. `BulletinA/2022-10-05/ser7.dat` -> `BulletinA/ser7.dat`)

This was removed because although it looks like versioning, we weren't actually utilizing it and we only do this for these two files. Removed for symmetry with how we handle other files.

Reply:
> This was removed because although it looks like versioning, we weren't actually utilizing it and we only do this for these two files. Removed for symmetry with how we handle other files.

Yeah I agree. That was also linked to the structure of the data for these in the data image we were using so far (parsing per date folders). Manifest update timestamp takes care of this 💯